packages/pug-lexer/index.js

```diff
@@ -971,6 +971,31 @@ Lexer.prototype = {
     }
   },
 
+  /**
+   * EachOf.
+   */
+
+  eachOf: function() {
+    var captures;
+    if (captures = /^(?:each|for) (.*) * of *([^\n]+)/.exec(this.input)) {
```

Comment:
```suggestion
    if (captures = /^(?:each|for) (.*) of *([^\n]+)/.exec(this.input)) {
```

The `.*` will already consume all the whitespace, which I think is fine.